SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1954,8 +1933,9 @@ void CNEMOEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container
   bool gravity      = config->GetGravityForce();
   bool implicit     = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
 
-  su2double *U_domain = new su2double[nVar];      su2double *U_outlet = new su2double[nVar];
-  su2double *V_domain = new su2double[nPrimVar];  su2double *V_outlet = new su2double[nPrimVar];
+  su2double *U_domain;
+  su2double *U_outlet = new su2double[nVar];
```

Comment:
For some reason, I can't do something without the regression values changing:
`su2double *U_outlet;
U_outlet = nodes->GetSolution(iPoint);
for (iVar =0 ....) U_outlet[iVar]=0.0;`

Any ideas?

Reply:
That would write over the solution at iPoint since GetSolution returns a pointer, that's where the "const everything" guideline comes from.
`const su2double* U_outlet = nodes->GetSolution(iPoint);`
With that said, maybe that is what you are trying to do?

Reply:
Ahh. My idea was to initialize the U_outlet vector with the 'new' command.

Ideally this problem goes away once I break the reliance of the conserved variables in some of the numerical schemes.